.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -1,34 +1,34 @@
 name: Bug report
 description: Use this template for reporting bugs. Please search existing issues first.
 labels: bug, needs-triage
+type: Bug
 body:
-  - type: markdown
+  - type: textarea
     attributes:
-      value: '## Detailed issue description'
+      label: Describe the bug
   - type: textarea
     attributes:
       label: Steps to reproduce the problem
     validations:
       required: true
   - type: textarea
     attributes:
-      label: Expected behaviour
+      label: Expected behavior
     validations:
       required: true
   - type: textarea
     attributes:
-      label: Actual behaviour
+      label: Actual behavior
     validations:
       required: true
-  - type: markdown
-    attributes:
-      value: '## Environment'
   - type: input
+    id: version
     attributes:
       label: Grafana k6 Studio version
     validations:
       required: true
   - type: input
+    id: os
```

Comment:
ids should allow us to pre-fill these fields automatically (coming in the next PR 👀)